### ISSUE TITLE
Master does not work anymore on azure AKS

### ISSUE THREAD

**Issue by quertenmont** [gramineproject/gsc#114]
Hello @dimakuv 

I am experiencing problems if I use master branch to deploy on azure AKS.
At runtime, my container fails with: "error: Cannot open device /dev/sgx_enclave. Please make sure the Intel SGX kernel module is loaded."

If I revert gsc to commit 7d5ca56cdac3caa33193138d90593f1c63a88199, then It woks perfectly.
In both case, my config.yaml contains:

```
SGXDriver:
    Repository: "https://github.com/intel/SGXDataCenterAttestationPrimitives.git"
    Branch:     "DCAP_1.11 && cp -r driver/linux/* ."
```

I also tried to keep the default in recent version, but it doesn't work better.
So I suspect compatibility with AKS was broken recently

**Comment by dimakuv** [comment]
Duplicate of #115.

**Comment by jkr0103** [comment]
Fixed with PR #117, hence can be closed.
